.travis.yml

```diff
@@ -66,6 +66,16 @@ jobs:
                 fi
             fi
 
+      env:
+        - NAME=prohibited-word-check
+      script:
+        # Run scripts to search for added prohibited words
+        - python check_tools/validate.py -c -i check_tools/prohibited.txt -f . &> check_tools/pr_black_list.log
+        - git remote add upstream https://github.com/armmbed/mbed-os-5-docs.git
+        - git fetch --depth=1 upstream development
+        - git checkout development
+        - python check_tools/validate.py -c -i check_tools/prohibited.txt -f . &> check_tools/head_black_list.log
```

Comment:
What's the reason that this will not always be empty?

Reply:
If a `check_tools` directory already exists, why not commit this `head_black_list.log` (whitelist?) with the developement branch as well?